process/sync/baseSync.go

```diff
@@ -240,9 +259,55 @@ func (boot *baseBootstrap) loadNotarizedBlocks(blockFinality uint64,
 		}
 	}
 
+	if currentNonce <= blockFinality {
+		return process.ErrBoostrapFromStorage
+	}
+
+	for i := currentNonce + 1; i <= highestNonceInStorer; i++ {
+		boot.cleanupNotarizedStorage(i, hdrNonceHashDataUnit)
```

Comment:
no need for this function. it could be done here.

Reply:
I just wanted to mentain the same format used in loadBlocks, just in case that in the future we would need to clean more stuff